requirements/requirements-python3.6.txt

```diff
@@ -43,7 +43,7 @@ apispec==1.3.3
 appdirs==1.4.3
 argcomplete==1.11.1
 asn1crypto==1.3.0
-astroid==2.3.3
+astroid==2.4.0
```

Comment:
I really think think we should update our process/automation to not touch these files when setup.py hasn't been touched either. (Ideally not unless the versions _in_ setup.py are updated.)

Reply:
I think these were a result of a rebase & force-push. The original commit did bump FAB version in setup.py. Hence needed this but I am sure this are not needed anymore

Reply:
Yes it was. And it's currently done as you expect @ashb -> every time 'generate requirements" is run, the "setupN.N.txt" is generated with md5hash of setup.py. So only after setup.py is changed, you should run the "generate-requirements" to regenerate requirements and the setup*.txt file. 

In fact even now if we regenerate the requirements always in the cron job, it is purely information in the logs - it does not have to be fixed until the next setup.py modification and the job never fails when requirement files are modified by it.  But it might be helpful if cron job will start failing because of dependency update - then we will see in the logs clearly which requirements were upgraded by the CRON job, so that we can pin-point the culprit easily and know how to fix it quickly.

In this case, it was indeed fully justified as there were two parallel modifications of the setup.py in two different branches. And failing build correctly protected us from potentially conflicting/wrong requirements. So I think all worked as expected.

Reply:
Makes sense, I thought that was what we did, I just hadn't seen the history of the PR so was a bit confused.

Thanks Jarek